addons/mail/data/mail_data.xml

```diff
@@ -68,5 +68,32 @@
             <field name="name">Discussions</field>
             <field name="sequence" eval="0"/>
         </record>
+
+        <record id="action_server_module_install_mail_send" model="ir.actions.server">
+            <field name="name">Module Immediate Install Mail Send</field>
+            <field name="condition">True</field>
+            <field name="type">ir.actions.server</field>
+            <field name="model_id" ref="base.model_ir_module_module" />
+            <field name="state">code</field>
+            <field name="code">module_id = context.get('active_id')
```

Comment:
I just checked and we cannot schedule mail in the future so this version is fine, keep it like that.